app/views/service_calendars/master_calendar/pppv/totals/_pppv_totals.html.haml

```diff
@@ -0,0 +1,24 @@
+-# Copyright © 2011-2016 MUSC Foundation for Research Development
```

Comment:
Renders the totals for an arm on initial load in `table.html.haml` and `table.js.coffee`